library/desktop/src/clients/menu.rb

```diff
@@ -66,6 +66,13 @@ def main
         "Hidden"
       ]
 
+      # always load the desktop files from the system root
+      if WFM.scr_chrooted?
+        # revert back the chrooting
+        handle = Yast::WFM.SCROpen("chroot=/:scr", false)
+        Yast::WFM.SCRSetDefault(handle)
+      end
```

Comment:
and will be the started modules chrooted or not?

Reply:
Yes, they will, it starts new subprocesses which have their own SCR instances.